### ISSUE TITLE
Upload matplotlib-3d to Pypi

### ISSUE THREAD

**Issue by aayrolles** [rougier/matplotlib-3d#5]
It should be helpful to deploy matplotlib-3d on PyPI, the Python Package Index.

**Comment by rougier** [comment]
Yes, I agree. I'll try to do that in the coming weks. Don't hesitate to remind here.
